frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -504,7 +510,7 @@ const PairElement = DragSource(
                 <input
                   type="text"
                   className="pf-c-form-control"
-                  placeholder={nameString.toLowerCase()}
+                  placeholder={nameKey ? nameKey : nameString.toLowerCase()}
```

Comment:
```suggestion
                  placeholder={nameKey || nameString.toLowerCase()}
```

But it would be better to not have two different ways to pass this string if possible.